src/tools/wasm-ctor-eval.cpp

```diff
@@ -1196,24 +1272,6 @@ EvalCtorOutcome evalCtor(EvallingModuleRunner& instance,
       return EvalCtorOutcome();
     }
   }
-
-  // Otherwise, we don't recognize a pattern that allows us to do partial
-  // evalling. So simply call the entire function at once and see if we can
-  // optimize that.
-
-  Literals results;
-  try {
-    results = instance.callFunction(funcName, params);
-  } catch (FailToEvalException& fail) {
-    if (!quiet) {
-      std::cout << "  ...stopping since could not eval: " << fail.why << "\n";
-    }
-    return EvalCtorOutcome();
-  }
-
-  // Success! Apply the results.
-  interface.applyToModule();
-  return EvalCtorOutcome(results);
```

Comment:
Where did this code go?

Reply:
We now treat functions with `block` bodies and non-block bodies uniformly by wrapping non-block bodies in blocks. See line 1098: `auto* block = builder.blockify(func->body);`. This made putting the whole thing in a loop much simpler.

Reply:
I must be missing something here, sorry. This code here does a try-catch. I was expected to see it replaced by another try-catch. That seems unrelated to the `blockify` issue you mention, unless there is a subtle connection I am missing.

Reply:
I see, thanks! Sorry I missed that, maybe I did control-F without expanding the diff enough...